net/natmap/Makefile

```
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=natmap
+PKG_VERSION:=20221202
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/heiher/natmap.git
+PKG_SOURCE_DATE:=2022-11-11
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=c41ee4a4525c59103e66ee96afac4a636728d714c3f6af4a25f38a71eff71bd9
```

```suggestion
PKG_NAME:=natmap
PKG_RELEASE:=1

PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/heiher/natmap
PKG_SOURCE_VERSION:=20221202
PKG_MIRROR_HASH:=c41ee4a4525c59103e66ee96afac4a636728d714c3f6af4a25f38a71eff71bd9
```

Also, you need to fix line 29.

---

Fixed.